patches/chrome-browser-signin-account_consistency_mode_manager.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/signin/account_consistency_mode_manager.cc b/chrome/browser/signin/account_consistency_mode_manager.cc
+index 0093a0e6f96ebe73f1c89d879aeb318ccd5e5e7e..1270ac9b5ac9f3a149aeb0a47bfb64467a7fddd0 100644
+--- a/chrome/browser/signin/account_consistency_mode_manager.cc
++++ b/chrome/browser/signin/account_consistency_mode_manager.cc
+@@ -182,6 +182,7 @@ AccountConsistencyModeManager::ComputeAccountConsistencyMethod(
+     Profile* profile) {
+   DCHECK(ShouldBuildServiceForProfile(profile));
+ 
++BRAVE_COMPUTE_ACCOUNT_CONSISTENCY_METHOD
```

Comment:
can we not set https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/signin/account_consistency_mode_manager.cc;l=196?q=account_consistency_mode_manager.cc&ss=chromium%2Fchromium%2Fsrc to false?
If it's hitting ENABLE_MIRROR first then let's set `enable_mirror` to `false` in gn

Reply:
I see the problem now with the `#error`, but maybe we can set `ENABLE_DICE_SUPPORT` to true for just the scope of this file